src/test/java/com/algosenpai/app/commands/VolumeCommandTest.java

```diff
@@ -40,7 +40,7 @@ void testSoundLevelWrongInput() throws IOException {
     }
 
     @Test
-    void testSoundLevelMissingInput() throws IOException {
+    void testSoundLevelMissingInput() throws Exception {
```

Comment:
Is there a reason for change from Exception to IOException?

Reply:
Because now there are 2 different exceptions throws so I was lazy to put both

Reply:
Okay, I believe it is not a huge violation if it is a test case.